src/main/resources/com/mysql/cj/LocalizedErrorMessages.properties

```diff
@@ -1095,15 +1095,33 @@ ClusterAwareReaderFailoverHandler.3=[ClusterAwareReaderFailoverHandler] Trying t
-ReadWriteSplittingPlugin.5=The driver attempted to update its topology information but did not find a matching host entry for the current connection in its host list. Please ensure you are using the clusterInstanceHostPattern if your connection string does not follow the standard RDS format.
-ReadWriteSplittingPlugin.6=setReadOnly(false) was called on a read-only connection inside a transaction. Please complete the transaction before calling setReadOnly(false).
-ReadWriteSplittingPlugin.7=An error occurred while syncing session state during a read/write split.
-ReadWriteSplittingPlugin.8=The 'clusterInstanceHostPattern' configuration property is required when an IP address or custom domain is used to connect to a cluster that provides topology information. If you would instead like to connect without failover functionality, set the 'enableClusterAwareFailover' configuration property to false.
-ReadWriteSplittingPlugin.9=The driver attempted to update its internal connection information but did not find a matching host entry for the current connection in its host list. Please ensure you are using the clusterInstanceHostPattern if your connection string does not follow the standard RDS format.ReadWriteSplittingPlugin.10=setReadOnly(false) was called inside a transaction while connected to a reader instance. Please commit/rollback the transaction before calling setReadOnly(false)
+ReadWriteSplittingPlugin.1=[ReadWriteSplittingPlugin] An error occurred while trying to switch to a reader connection.
+ReadWriteSplittingPlugin.2=[ReadWriteSplittingPlugin] An error occurred while trying to switch to a writer connection.
+ReadWriteSplittingPlugin.3=[ReadWriteSplittingPlugin] The driver was not able to retrieve any host information. Please ensure your connection string is correct.
+ReadWriteSplittingPlugin.4=[ReadWriteSplittingPlugin] setReadOnly(false) was called on a read-only connection inside a transaction. Please complete the transaction before calling setReadOnly(false).
+ReadWriteSplittingPlugin.5=[ReadWriteSplittingPlugin] An error occurred while syncing session state during a read/write split.
+ReadWriteSplittingPlugin.6=[ReadWriteSplittingPlugin] The 'clusterInstanceHostPattern' configuration property is required when an IP address or custom domain is used to connect to a cluster that provides topology information.
+ReadWriteSplittingPlugin.7=[ReadWriteSplittingPlugin] The driver attempted to update its internal connection information but did not find a matching host entry for the following host in its host list: ''{0}''. Please ensure you are using the clusterInstanceHostPattern if your connection string does not follow the standard RDS format.
+ReadWriteSplittingPlugin.8=[ReadWriteSplittingPlugin] Transaction boundary detected - switching to a new reader instance.
+ReadWriteSplittingPlugin.9=[ReadWriteSplittingPlugin] {0} = {1}
+ReadWriteSplittingPlugin.10=[ReadWriteSplittingPlugin] {0} = ''{1}''
```

Comment:
These two logs seem to have similar usage, do we need both of them?

Reply:
Not necessarily, I used the first one for boolean values and the second for string values. For example:
- First log: `loadBalanceReadOnlyTraffic = true`
- Second log: `clusterInstanceHostPattern = 'my-host.aws.com'`

I can potentially get rid of the first one if you think that would be better, just means the boolean value would be in quotes